packages/cli/lib/services/response-saver.service.ts

```diff
@@ -10,12 +10,29 @@ export function ensureDirectoryExists(directoryName: string): void {
     }
 }
 
-export function saveResponse<T>({ directoryName, data, customFilePath }: { directoryName: string; data: T; customFilePath: string }): void {
+function saveResponse<T>({
+    directoryName,
+    data,
+    customFilePath,
+    concatenateIfExists
+}: {
+    directoryName: string;
+    data: T | T[];
+    customFilePath: string;
+    concatenateIfExists: boolean;
+}): void {
     ensureDirectoryExists(`${directoryName}/mocks`);
 
     const filePath = path.join(directoryName, customFilePath);
     ensureDirectoryExists(path.dirname(filePath));
 
+    if (fs.existsSync(filePath)) {
+        const existingData = JSON.parse(fs.readFileSync(filePath, 'utf8'));
```

Comment:
hopefully we won't have too big saved response because loading/concat/rewrite could become quite slow